xml/book_cloud_suppl.xml

```diff
@@ -881,6 +881,82 @@
     <link xlink:href="http://docs.openstack.org/image-guide/content/image-metadata.html"/>.
    </para>
   </sect1>
+  <sect1 xml:id="sec.suppl.img.custom.kernel">
+   <title>Using a Custom Kernel to Boot Btrfs Images Under &xen;</title>
+
+   <para>
+    On &xen; hosts, starting instances from an image that uses Btrfs as root
+    file system may fail with &productname; &productnumber;. As a work-around,
+    boot the Btrfs image with a custom kernel to start the instances. Prepare
+    the Btrfs image as described in
+    <xref
+     linkend="pro.suppl.img.brtfs.custom.kernel"/>.
+   </para>
+
+   <procedure xml:id="pro.suppl.img.brtfs.custom.kernel">
+    <title>Booting Btrfs Images with a Custom Kernel</title>
+    <para>
+     Execute the following on your &contrnode;:
```

Comment:
Thanks! Added this to the requirements for the procedure (apart from grub.xen, which I decided to mention in step 1 where the ```grub2-xen``` package is installed).